### DISCUSSION TITLE
Bacnet Client for ESP

### DISCUSSION THREAD

**Discussion by zachary-y-negawatt** [bacnet-stack/bacnet-stack#626] [Q&A]
I want to know where can find a bacnet client demo?
I am going to a readProperty function for reading the data from bacnet IP, but I don't know how to get the value by IP or ethernet port by the library

My idea like this JS code. 
```
const bacnet = require('node-bacnet');
const client = new bacnet();

client.readProperty('192.168.1.43', {type: 8, instance: 44301}, 28, (err, value) => {
  console.log('value: ', value);
});
```

**Comment by skarg** [comment]
Perhaps you should use [Node BACstack](https://www.npmjs.com/package/bacstack) - A BACnet protocol stack written in pure JavaScript.

**Reply by zachary-y-negawatt** [reply]
Thank you your suggestion
